File: mcap-quartz/src/main/java/com/marshal/mcap/quartz/service/JobRunningInfoService.java
package com.marshal.mcap.quartz.service;

import com.marshal.mcap.quartz.entity.JobRunningInfo;
import com.marshal.mcap.core.beans.ResponseData;

import java.util.List;

/**
 * Created with IntelliJ IDEA.
 * author: Marshal
 * Date: 2018/11/1
 * Time: 21:57
 * Description:
 */
public interface JobRunningInfoService {

    List<JobRunningInfo> queryJobRunningInfo(JobRunningInfo condition, int pageNum, int pageSize);

    void saveJobRunningInfo(JobRunningInfo jobRunningInfo);
}

File: halcyon-web/src/main/resources/static/js/service/quartz/jobRunningInfoService.js
/**
 * author:Marshal
 * description:service层,与数据库交互
 */
app.service("jobRunningInfoService",function($http){
    //base crud
    this.query=function(pageNum,pageSize,condition){
        return $http.post("../quartz/jobRunningInfo/query?pageNum="+pageNum+"&pageSize="+pageSize,condition);
    }
});